src/bundle/Resources/views/content/modal_draft_conflict.html.twig

```diff
@@ -22,29 +22,37 @@
                         ) }}
                     </p>
                     <p class="ez-modal-body__main-content">
-                        {{ 'draft.conflict.choice'|trans({})|desc('You can either edit any of your existing draft(s) or add a new one.') }}
+                        {% if content_is_user %}
+                            {{ 'draft.conflict.choice_content_is_user'|trans({})|desc('You can edit any of your existing draft(s)') }}
+                        {% else %}
+                            {{ 'draft.conflict.choice'|trans({})|desc('You can either edit any of your existing draft(s) or add a new one.') }}
+                        {% endif %}
                     </p>
                 </div>
-                {% include '@ezdesign/parts/table_header.html.twig' with { headerText: null, tools: version_modal_draft_conflict.table_header_tools() } %}
+                {% set show_add_draft_button = (content_is_user != true) %}
```

Comment:
nitkpick: `content_is_user is not true`?

Reply:
I think I tried all combinations of those and for some reason this is the only one not resulting in syntax error.